templates/emails/thankyou.kendraio.hbs

```diff
@@ -1,77 +0,0 @@
-{{#if interval }}
-Subject: Thank you for your {{currency donation.amount currency=donation.currency}}/{{interval}} contribution to {{group.name}}
```

Comment:
This :arrow_up: `group` was renamed to `collective` [in 2017](https://github.com/opencollective/opencollective-api/blob/0335e830f09ebbd89fa37d34531ce1e8a0267c67/migrations/archives/201707140000-GroupToCollective.js)